tests/phpunit/tests/test-links-domain-sub-dir.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @group links
+ * @group domain
+ * @group directory
+ */
+class Links_Domain_Sub_Dir_Test extends PLL_Domain_UnitTestCase {
+	use PLL_Links_Trait;
+
+	protected $is_directory = true;
+
+	public function set_up() {
+		parent::set_up();
+
+		$this->hosts = array(
+			'en' => 'http://example.org',
+			'fr' => 'http://example.fr',
+			'de' => 'http://example.de',
+		);
+
+		self::$model->options['default_lang'] = 'en';
```

Comment:
This one is useless now, it is set in `PLL_Domain_UnitTestCase`.
```suggestion
```

Reply:
Damn!